src/extensions/replay/sessionrecording.ts

```diff
@@ -392,6 +392,10 @@ export class SessionRecording {
             return 'disabled'
         }
 
+        if (this._urlBlocked) {
+            return 'paused'
+        }
```

Comment:
the fix is to have this as the earliest check in status
rrweb continues to emit, because it doesn't know that it is paused
and we continue to check
until the URL changes and we resume